assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/index.js

```diff
@@ -61,7 +61,10 @@ function DashboardAllTrafficWidget( props ) {
 			return true;
 		}
 
-		return select( CORE_USER ).canViewSharedModule( 'analytics' );
+		return (
+			select( CORE_USER ).canViewSharedModule( 'analytics' ) ||
+			select( CORE_USER ).canViewSharedModule( 'analytics-4' )
+		);
```

Comment:
I understand why `canViewSharedAnalytics` was updated for  `DashboardViewIndicator`, great catch there, thank you!

To confirm, is it necessary to check if `analytics-4` is a viewable shared module within the UA dashboard view specific dashboard widget components, provided that `canViewSharedAnalytics` here is only used as a prerequisite for calling `modules/analytics` selectors?

Reply:
That's a great point, these settings are mirrored so no. I'll remove them 👍🏻